ci/blackbox/rdsbroker_test.go

```diff
@@ -381,3 +402,82 @@ func pollForOperationCompletion(instanceID, serviceID, planID, operation string)
 	fmt.Fprintf(GinkgoWriter, "done. Final state: %s.\n", state)
 	return state
 }
+
+type bindingResponse struct {
+	Credentials rdsbroker.Credentials `json:"credentials"`
+}
+
+func getCredentialsFromBindResponse(resp *http.Response) (*rdsbroker.Credentials, error) {
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	bindingResponse := bindingResponse{}
+	err = json.Unmarshal(body, &bindingResponse)
+	if err != nil {
+		return nil, err
+	}
+
+	return &bindingResponse.Credentials, nil
+}
+
+func setupPermissionsTest(databaseUri string) error {
+	dbURL, err := url.Parse(databaseUri)
+	if err != nil {
+		return err
+	}
+
+	db, err := sql.Open("postgres", dbURL.String())
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	_, err = db.Exec("CREATE TABLE " + permissionCheckTableName + " (id integer)")
+	if err != nil {
+		return fmt.Errorf("Error creating table: %s", err.Error())
+	}
+
+	_, err = db.Exec("INSERT INTO " + permissionCheckTableName + " VALUES(42)")
+	if err != nil {
+		return fmt.Errorf("Error inserting record: %s", err.Error())
+	}
+
+	return nil
+}
+
+func permissionsTest(databaseUri string) error {
+	dbURL, err := url.Parse(databaseUri)
+	if err != nil {
+		return err
+	}
+
+	db, err := sql.Open("postgres", dbURL.String())
+	if err != nil {
+		return err
+	}
+
+	defer db.Close()
+
+	// Can we write?
+	_, err = db.Exec("INSERT INTO " + permissionCheckTableName + " VALUES(43)")
+	if err != nil {
+		return fmt.Errorf("Error inserting record: %s", err.Error())
+	}
+
+	// Can we ALTER?
+	_, err = db.Exec("ALTER TABLE " + permissionCheckTableName + " ADD COLUMN something INTEGER")
+	if err != nil {
+		return fmt.Errorf("Error ALTERing table: %s", err.Error())
+		return err
```

Comment:
That's a good point actually, we can probably remove those endpoints from the healthcheck app now as they're no longer used.